actions/hook.go

```diff
@@ -29,10 +39,10 @@ var hooks = map[HookType]NewHookFunc{
 
 var ErrUnknownHookType = errors.New("unknown hook type")
 
-func NewHook(h HookType, a *Action, ah ActionHook) (Hook, error) {
-	f := hooks[h]
+func NewHook(a *Action, ah ActionHook) (Hook, error) {
+	f := hooks[HookType(ah.Type)]
```

Comment:
I've also updated the type to match HookType

Reply:
Saw that. Still not in master, right?

Reply:
right :(